### ISSUE TITLE
Two heading level 1 is present in hello world page

### ISSUE THREAD

**Issue by Karan4495** [dotnet/maui#6261]
GitHubTags:
#.NET MAUI; #A11ySev2; #A11yMAS; #A11yTCS; #Benchmark; #DesktopApp; #Win11; #WCAG2.4.6; #GH_.NETMAUI_Win32_Apr2022; #Headings and Labels​;

### Description

Two heading level 1 is present in hello world page

### Steps to Reproduce

1.Open .Net MAUI application solution Visual studio Enterprise 2022 Preview.
2.Tab till "MauiApp1" in solution explore and press enter.
3.Tab till Accept button and press enter.
4.Tab till Debug Target drop down to select Framework and then the net6.0-windows entry.
5.Tab till Windows Machine button and press enter.
6..Net MAUI template screen will open.
7.Verify Two heading level 1 is present in hello world page or not.

https://user-images.githubusercontent.com/100186395/164150675-e6635fd6-d029-4fc6-8c78-1da2592c22ec.mp4

### Version with bug

Unknown/Other (please specify)

### Last version that worked well

Unknown/Other

### Affected platforms

Windows

### Affected platform versions

Windows 11 Enterprise Version 21H2 OS build 22000.613

### Relevant log output

```shell
In hello world page should be only one heading level 1 be present.
```

**Comment by Karan4495** [comment]
Verified issue in ".NET MAUI Template". issue is fixed so closing the bug
Windows 11 Enterprise Version 21H2 OS build 22000.675
Visual Studio Enterprise 2022 Int preview 17.3.0 Preview 2.0 [32513.392.main]
added attachment for references

https://user-images.githubusercontent.com/100186395/168576287-b763d4ea-c922-45c0-8ad5-fba32d650f71.mp4
